modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java

```diff
@@ -993,6 +1035,11 @@ private Metas getOrAllocateCacheMetas() throws IgniteCheckedException {
                     null,
                     null);
             }
+            catch (Throwable e) {
+                e.printStackTrace();
```

Comment:
Use `log`, please

Reply:
thanks, removed it - was used for debugging